desktop-widgets/divelistview.cpp

```diff
@@ -645,7 +645,12 @@ void DiveListView::addDivesToTrip()
 	std::vector<dive *> dives = getDiveSelection();
 	if (!t || dives.empty())
 		return;
-	Command::addDivesToTrip(QVector<dive *>::fromStdVector(dives), t);
+#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
```

Comment:
That looks surprisingly ugly.

What would you think about a general function of the kind
```
template<typename T>
QVector<T> stdToQt(const std::vector<T> &v)
{
#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
	return QVector<T>(v.begin(), v.end());
#else
	return QVector<T>::fromStdVector(c);
#endif
}
```
in `command_base.h`?
Totally untested, but something like this should work. We could extend this to other types, if needed.